js/localization/messages/en.json

```diff
@@ -155,12 +157,15 @@
         "dxDataGrid-ariaSearchInGrid": "Search in data grid",
         "dxDataGrid-ariaSelectAll": "Select all",
         "dxDataGrid-ariaSelectRow": "Select row",
+        "dxDataGrid-ariaToolbar": "Data grid's toolbar",
```

Comment:
```suggestion
        "dxDataGrid-ariaToolbar": "DataGrid's toolbar",
```

Reply:
This text is used by screen reader. There is no word 'DataGrid' in English so we use 'data grid' and 'tree list' in our aria-labels.